packages/compass-import-export/src/components/import-preview.tsx

```diff
@@ -60,7 +59,7 @@ function ImportPreview({
     type: string;
     checked: boolean;
   }[];
-  values: Document[];
+  values: string[][];
```

Comment:
The type was wrong before. Noticed when I wrote code to turn an array of string arrays into an array of docs and it didn't work.

The preview values are an array of up to length 10 where each item in the array is an array of strings which represent the CSV values in that row.